### DISCUSSION TITLE
How to run custom action before uptime check?

### DISCUSSION THREAD

**Discussion by madhogs** [upptime/upptime#742] [Q&A] [answered]
Most of my sites are not publicly available and require being connected to my private network using tailscale.
Tailscale have a github action available which will connect a workflow to a tailscale network - https://github.com/marketplace/actions/connect-tailscale

I can add a step before the update step in uptime.yml but it says not to edit this file directly.
Is there anywhere/anyway to add an action in such a way that it won't be removed when the template updates?

Thanks

**Comment by AnandChowdhary** [comment, marked as answer]
Unfortunately this is currently unsupported because if you edit the `.upptime.yml` file directly, it will be overwritten in the next update. :(
